contrib/docker/Dockerfile

```diff
@@ -12,7 +21,7 @@ RUN go mod download
 RUN go mod download -modfile=.bingo/golangci-lint.mod
 
 COPY . ./
-RUN rm -rf bin
+#RUN rm -rf bin
```

Comment:
The need to delete the bin folder is to make sure that we are re-building/downloading the binaries using .bingo. when we build the container image and not use the downloaded ones from git

Reply:
I think a better way to get rid of the bin directory in the docker image is to add a `.dockerignore` file to the repository with a line to exclude `bin/`.
https://docs.docker.com/engine/reference/builder/#dockerignore-file